SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.UWP/Utils/ProcessInformation.cs

```diff
@@ -19,6 +19,10 @@ class ProcessInformation : IProcessInformation
 
         public string ParentProcessName => ProcessDiagnosticInfo.GetForCurrentProcess().Parent?.ExecutableFileName;
 
-        public string ProcessArchitecture => Package.Current.Id.Architecture.ToString();
+        /// <remarks>
+        /// ARM64 was added to ProcessorArchitecture enum (that can be received by Package.Current.Id.Architecture call) only in SDK version 18362, 
+        /// so casting to string is incorrect on lower versions.
+        /// </remarks>
+        public string ProcessArchitecture => RuntimeInformation.ProcessArchitecture.ToString();
```

Comment:
This change "trims" functionality for .NET framework versions `4.7 4.6.2 4.6.1 4.6 4.5.2 4.5.1 4.5 4.0 3.5 3.0`, won't there be any conflicts?

Reply:
This is from `Microsoft.NETCore.UniversalWindowsPlatform` that we require via NuGet. Also, this is only for UWP. I don't see what kind of conflicts you are expecting